### PR TITLE
Import from CSV file

### PR DESCRIPTION
**Pull request by macejmic**
closes https://github.com/3scale/3scale_toolbox/issues/7

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@
 /pkg/
 /spec/reports/
 /tmp/
+.DS_Store
```
Comment (mikz) [inline .gitignore]: this really should be in your global gitignore :) but i'm with it it here too

File: examples/import.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+## Import from a CSV file
```
Comment (mikz) [inline examples/import.md]: this should not be in `examples` folder no ? but rather `docs` ?

File: exe/3scale-import [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+
+  services.keys.each do |service_name|
+    # create service
+    service = client.create_service name: service_name
+
+    if service['errors'].nil?
+      stats[:services] += 1
+      puts "Service #{service_name} has been created."
+    else
+      raise "Service has not been saved. Errors: #{service['errors']}"
```
Comment (mikz) [inline exe/3scale-import]: might be better to `abort`. there is no need for stacktrace in the cli app

**Comment by mikz** [comment]
Like it! I think it is totally fine for now.

File: .gitignore (hunk 1 above)
Comment (macejmic) [inline .gitignore]: done :)

File: docs/import_example.csv [hunk 4]
```diff
@@ -0,0 +1,7 @@
+service_name,endpoint_name,endpoint_http_method,endpoint_path,auth_mode,endpoint_system_name,type
```
Comment (mikz) [inline docs/import_example.csv]: this one can be in `examples` folder :)

**Comment by andrewdavidmackenzie** [comment]
+1 for including example files.

For this toolbox, do we need to plan some "self-documentation" (help option? man page? other?) for people who will install the gem from a gem server?

**Comment by mikz** [comment]
@andrewdavidmackenzie the plan is that it should be self describing. Run the command without anything , get help. Run one of those without params, get help. Etc. Like any other CLI tool there.

Man page makes sense also. But I'd say it is not a priority for now.

**Pull request merged**